src/sentry/static/sentry/app/components/pluginConfig.jsx

```diff
@@ -81,6 +81,23 @@ const PluginConfig = React.createClass({
     });
   },
 
+  testPlugin() {
+    let loadingIndicator = IndicatorStore.add(t('Sending test..'));
+    this.api.request(this.getPluginEndpoint(), {
+      method: 'POST',
+      data: {
+        test: true
+      },
+      success: data => {
+        IndicatorStore.remove(loadingIndicator);
+        IndicatorStore.add(t('Test Complete!'), 'success');
```

Comment:
it doesn't seem like you're actually displaying the test results anywhere. you should save the results in state and display them somewhere in the page. i think this is what we were doing before the move to react: https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/plugins/project_configuration.html